ush/setup.py

```diff
@@ -215,6 +218,12 @@ def add_jobname(tasks):
     # stranglers.
     update_dict(cfg_d, cfg_d)
 
+    # Load one more if running Coupled AQM
+    if cfg_d['cpl_aqm_parm']['CPL_AQM']:
+        cfg_aqm = get_yaml_config("config_defaults_aqm.yaml")
+        update_dict(cfg_aqm, cfg_d)
```

Comment:
Here's where the `CPL_AQM = True` magic happens now, instead of in the run script.

Reply:
Baby steps, but all those `update_dict()` calls look like something `uwtools` ought to be doing in a much more concise way. A statement like `update_dict(cfg_d, cfg_d)` boggles the mind. Is some kind of dereferencing being done there? Seeing this use case -- and assuming I'm understanding it at all -- I'm wondering about frictionless we might make this in `uwtools`. I'll bring it up in Slack.

Reply:
With uwtools, I've done something similar in the MPAS App that looks more like this:

```
    for supp_config in (platform_config, user_config):
        experiment_config.update_values(supp_config)
```

That seems much nicer. With the latest in uwtools, it would read:

```
    for supp_config in (platform_config, user_config):
        experiment_config.update_from(supp_config)
```